photon/content.md

```diff
@@ -0,0 +1,23 @@
+# VMware Photon OS
+
+%%LOGO%%
+
+Photon OS is a technology preview of a minimal Linux container host. It is designed to have a small footprint and boot extremely quickly on VMware platforms. Photon OS is intended to invite collaboration around running containerized applications in a virtualized environment.
+
+Photon contains `tdnf`, a new, open-source, yum-compatible package manager that will help make the system as small as possible, but preserve the robust yum package management capabilities.
```

Comment:
Guys around the office said you should've named it "cannon" or "torpedo" instead -- total missed opportunity here. :smile: